### ISSUE TITLE
U.S Virgin Islands order Allowed, but not in Allowed Countries

### ISSUE THREAD

**Issue by chugginn** [amzn/amazon-payments-magento-plugin#214]
We had an order placed through Amazon Payments for a resident in the U.S. Virgin Islands, but this country is not in our 'Allow Countries' list.

**Comment by dbrent-amazon** [comment]
I've seen this before. For me the issue was that a buyer added an address but "United States" was still selected in the drop-down when submitting. There is no zip code/address validation when adding a new address from the widgets.

See if you can reproduce by doing the following.

* Click "+ Add new" on the shipping widget. 
* Add a shipping address for a country you do not ship to.
* Leave "Country" set to "United States".
* Once the widget has been updated with the new shipping address Magento will still show shipping quotes.

**Comment by chugginn** [comment]
I attempted to replicate the order exactly as the customer had entered their details. When United States was entered as the country the order came in with a US country and shipping was applied normally as if shipping to a US state. When I chose U.S. Virgin Islands from the country dropdown, there were no shipping methods available, just our custom error message, but I was still allowed to place an order. Screenshot of shipping method attached.
![ship](https://cloud.githubusercontent.com/assets/2301471/10464693/bb861efa-71b0-11e5-8e66-648569a9d900.JPG)
